### DISCUSSION TITLE
Daily 12/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#55] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana estuve realizando los ejercicios pendientes de la plataforma y leyendo los materiales que se encuentran en la misma. El sabado participe de la planning que realizamos de manera sincrónica. Ayer me estuve ocupando de otras cuestiones así que no pude dedicarle tiempo al tema.
Hoy también estuve complicada, sólo pude leer unas páginas del nuevo material. Por la tarde participaré de la clase sincrónica.

**Comment by BraianTroncoso** [comment]
Hoy participé de la clase de Testing, y sigo capacitandome sobre android por mi cuenta. También estoy practicando un poco de Java. 
Paralelamente sigo capacitandome con otras cosas vinculadas.

**Comment by Valentino399** [comment]
Hoy estuve presente en la clase de Testing y estuve diseñando la home principal de la aplicación en Figma
